crates/bevy_asset/src/transformer.rs

```diff
@@ -241,3 +243,37 @@ impl<'a, A: Asset> TransformedSubAsset<'a, A> {
         self.labeled_assets.keys().map(|s| &**s)
     }
 }
+
+/// An identity [`AssetTransformer`] which infallibly returns the input [`Asset`] on transformation.]
+pub struct IdentityAssetTransformer<A: Asset> {
```

Comment:
This should be linked to in the `LoadTransformAndSave` docs :)

Reply:
Not sure how I feel about having this considering deprecating this pattern is kind of the reason we're deprecating LoadAndSave in the first place.... but ehh

Reply:
Agreed! I see `IdentityAssetTransformer` as a necessary evil. As @alice-i-cecile mentioned, file format conversion is a valid use-case for this transformer, and it does massively simplify the migration guide.